package.json

```
@@ -7,7 +7,9 @@
     "scripts": {
         "build:prod": "build-tools build --production --once",
         "build:dev": "build-tools build --once",
-        "watch:dev": "build-tools build"
+        "watch:dev": "build-tools build",
+        "test:e2e": "wp-env start && wp-scripts test-playwright",
+        "test:e2e:debug": "wp-env start && wp-scripts test-playwright --debug"
```

it's often useful to have the ui available. whilst it's possible to do so by running `npm run test:e2e -- --ui`, having a command for that would be nice
```suggestion
        "test:e2e": "wp-env start && wp-scripts test-playwright",
        "test:e2e:ui": "wp-env start && wp-scripts test-playwright --ui",
        "test:e2e:debug": "wp-env start && wp-scripts test-playwright --debug"
```